### PR TITLE
Allow passing length in hex with 0x notation

### PR DESCRIPTION
**Pull request by Qyriad**
This interprets a `0x` in front of the value passed with `--length`/`-n` as a hexadecimal number.

### CODE REVIEW

**Comment by ErichDonGubler** [comment]
Definitely related to #44. :)

**Comment by sharkdp** [comment]
@Qyriad Thank you very much for your contribution!

We will need to bump the minimum required Rust version (in `.travis.yml` and the `README`) in order for CI to pass.

File: src/main.rs [hunk 1]
```diff
@@ -258,7 +258,14 @@ fn run() -> Result<(), Box<::std::error::Error>> {
 
     if let Some(length) = matches
         .value_of("length")
-        .and_then(|n| n.parse::<u64>().ok())
+        .and_then(|n| {
+            if n.starts_with("0x") {
```
Comment (sharkdp) [inline src/main.rs]: We should probably also adapt the help-text to make users aware of this possibility.

**Comment by ErichDonGubler** [comment]
Pinging @Qyriad to see if they're still able to attend to this.

**Comment by Qyriad** [comment]
Oh right; yeah I can do this in the next day or two.

**Comment by sharkdp** [comment]
Sorry, I've missed your update. Could you please rebase this on `master` to resolve the conflicts? Thank you!

**Comment by Qyriad** [comment]
Done.

**Comment by sharkdp** [comment]
Thank you!

**Pull request merged**